python/tests.py

```diff
@@ -14,7 +15,7 @@ def _json_config_with_array(self):
 
     @property
     def _shared_file_path(self):
-        return 'config.json'
+        return os.path.normpath('../config.json')
```

Comment:
Should probably roll this back, since the rakefile calls the test from the root directory instead of the test directory.